Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -283,6 +285,7 @@
   components:
   - type: StorageFill
     contents:
+      - id: ClothingUniformJumpsuitKGBFemale
```

```suggestion
      - id: ClothingUniformJumpsuitKGBFemale # DS14-Soyuz
```
И перенести в конец списка